CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# How to contribute
```

Comment:
probably capitalise important words in titles, not just proper nouns.

Reply:
It's a stylistic choice and either is fine, as long as its consistent. mariadb.org, Wikipedia use sentence case for section headings (Bug reports), the KB was recently changed to use title case (Bug Reports). Even before this there was no consistency in README.md. I have no preference, so just pick one, and use it consistently.